radius.py

```diff
@@ -479,12 +479,31 @@ def handle_dashboard(rocket):
 """.strip()
 
 
+register_response = '''
```

Comment:
tiny nit: s/'''/"""

if you'd prefer, an alternative is to do treewide s/"""/'''